acceptance/tests/face/loadable_from_modules.rb

```diff
@@ -14,6 +14,12 @@
 initialize_temp_dirs
 
 agents.each do |agent|
+    
+  if (on(agent, facter("find fips_enabled")).stdout =~ /true/)
```

Comment:
I think this is querying for two facts named `find` and `fips_enabled`:

```
$ bx facter find virtual
find => nil
virtual => physical
```

I think you can just drop the `find` part. You might be thinking of `puppet facts find` though that returns all the facts.

Reply:
There is one test that used the find that I copied but anyway I changed it